tests/failing-symbolic.haskell

```diff
@@ -1,5 +1,7 @@
 tests/specs/benchmarks/ecrecoverloop00-sig1-invalid-spec.k
 tests/specs/bihu/collectToken-spec.k
+tests/specs/erc20/ds/transferFrom-failure-2-a-spec.k
+tests/specs/erc20/hkg/transferFrom-failure-2-spec.k
```

Comment:
We don't want to add to this list. It's OK to add to `tests/failing-symbolic.pyk`.

This list says "this proof doesn't work at all", and the other list `tests/failing-symbolic.pyk` says "this proof doesn't work with the pyk RPC based prover, but does work with the old prover".

So if you add this to the other list `tests/failing-symbolic.pyk`, and it's then passing (which you can test by removing from this list and adding to that list and running `make tests/specs/erc20/ds/transferFrom-failure-2-a-spec.k.prove`, for example), then we're good. But making the test fail altogether, we should isolate the failure reason and make a run-lemma-donel-lemma style claim for it and add the needed lemmas to get it passing with your new lemmas.